src/main/resources/languages/en.yml

```
@@ -52,6 +52,45 @@ CMD_Cannot_Distill: '&cThose ingredients cannot be distilled'
 CMD_Distill_Ruined: '&eThe brew was ruined in the distillation process'
 CMD_Age_Ruined: '&eThe brew was ruined in the aging process'
 
+# Defects
+Defect_WrongIngredient:
+  - 'There''s something in here that shouldn''t be in here...'
+  - 'One of the ingredients doesn''t taste good...'
+Defect_MissingIngredient:
+  - 'Seems like something''s missing...'
+  - 'Needs something more...'
+Defect_LowCount:
+  - 'You can barely taste one of the ingredients...'
+  - 'The flavor of one of the ingredients is too weak...'
+Defect_HighCount:
+  - 'One of the ingredients is overpowering...'
+  - 'The flavor of one of the ingredients is too strong...'
+Defect_NeedsDistill:
+  - 'You can''t taste the alcohol...'
+  - 'This needs some distillation...'
+Defect_BadDistill:
+  - 'The brew evaporated...'
+  - 'This definitely didn''t need distillation...'
+Defect_Uncooked:
+  - 'It''s still raw...'
+  - 'Not enough time in the oven...'
+Defect_Overcooked:
+  - 'It''s burnt...'
+  - 'Too much time in the oven...'
+Defect_UnderAged:
+  - 'This needs fermentation...'
+  - 'Not enough time in the barrel...'
+Defect_OverAged:
+  - 'This brew smells expired...'
```

```suggestion
  - 'This brew smells like vinegar...'
```
(Usually what happens when alcohol gets old, from what I know at least)